jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
I'm not sure why you don't like to use `sample_darknet_object_detection.launch`, is that your reason? ↑